_posts/2016-04-22-next-cio.md

```diff
@@ -0,0 +1,72 @@
+---
+published: true
+layout: post
+author: Adam Scott and Ashwin Vasan
+title: "We're searching for our next CIO"
+permalink: /articles/searching-for-our-next-cio/
+---
+
+![desk illustration](/img/cio/CIO_illustration.png)
+
+*The CFPB is looking for our next Chief Information Officer (CIO) to lead our technology team in supporting the mission of the CFPB and in setting an exceptional example for how technology can work in government.*
+
+The [Consumer Financial Protection Bureau (CFPB)](http://consumerfinance.gov) is a government agency that helps consumer finance markets work for Americans by making rules more effective, by consistently and fairly enforcing those rules, and by empowering consumers to take more control over their economic lives.  From the Bureau’s founding in the wake of the 2008 financial crisis, it has always been clear that we need to leverage modern technology and take a data-driven approach to achieve these goals.  The CFPB technology team uses and produces [open source code](https://github.com/cfpb), has embraced systems in the public cloud, and follows an [Agile](http://www.agilemanifesto.org/) software development model. Our team is comprised of talented individuals from both the public and private sectors, and through CFPB’s T[Technology and Innovation Fellows](http://www.consumerfinance.gov/jobs/technology-innovation-fellows/) program, we bring together designers, developers, data scientists, and security professionals to collaborate on building CFPB technology.
```

Comment:
Extra T before the link: "CFPB’s T[Technology and Innovation Fellows](http://www.consumerfinance.gov/jobs/technology-innovation-fellows/)"